src/symbol/shaping.js

```diff
@@ -35,7 +35,7 @@ export type PositionedGlyph = {
 
 // A collection of positioned glyphs and some metadata
 export type Shaping = {
-    positionedGlyphs: Array<PositionedGlyph>,
+    positionedGlyphs: {[number]: Array<PositionedGlyph>},
```

Comment:
Is it important for the key to be any number or can this be an array of arrays instead? It seems like using an array might simplify some things elsewhere like the ` for (const lineIndex of Object.keys(positionedGlyphs).map(Number)) {` in quads.js